### PR TITLE
Clean up remainder of PA-App and AW-App removal stuff

### PR DESCRIPTION
**Pull request by aberonni**
[AB#27894](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/27894) 

I basically searched for the following terms in the codebase and deleted/updated accordingly:
- pa app
- pa-app
- paApp
- aw app
- aw-app
- awApp

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I have communicated above whether I need any deviation from our PR guidelines (eg. "I don't want the reviewer to merge on my behalf because...")

### CODE REVIEW

File: interfaces/Portal/src/assets/i18n/nl.json [hunk 1]
```diff
@@ -247,7 +247,7 @@
           "inclusionEnded": "Hulpvragers waarvoor de opname is beëindigd. <br> Zij ontvangen geen (meer) betalingen voor de volgende betalings-/uitbetalingsronde.",
           "invited": "Hulpvragers die zijn uitgenodigd (na import).",
           "noLongerEligible": "De hulpvragers die werden gemarkeerd als 'niet langer in aanmerking komend'. <br> Dit zijn hulpvragerss die waren uitgenodigd maar zich nog niet hadden geregistreerd en inmiddels niet meer in aanmerking komen.",
-          "registered": "Hulpvragers die hun registratie in de PA-app hebben voltooid.",
+          "registered": "Hulpvragers die hun registratie hebben voltooid.",
```
Comment (aberonni) [inline interfaces/Portal/src/assets/i18n/nl.json:250]: Is het goed in het Nederlands?

File: services/121-service/src/fsp/dto/update-fsp.dto.ts [hunk 2]
```diff
@@ -61,7 +61,7 @@ export class CreateFspAttributeDto extends UpdateFspAttributeDto {
 
 export class UpdateFspDto {
   @ApiProperty({
-    example: { en: 'FSP PA-app display name', nl: 'FSP PA-app weergavenaam' },
+    example: { en: 'FSP display name', nl: 'FSP weergavenaam' },
```
Comment (aberonni) [inline services/121-service/src/fsp/dto/update-fsp.dto.ts:64]: I'm not confident about this change but this seemed to be the case from some (shallow) searching.

File: services/121-service/src/programs/programs.controller.ts [hunk 3]
```diff
@@ -56,6 +56,7 @@ export class ProgramController {
     private readonly koboConnectService: KoboConnectService,
   ) {}
 
+  // XXX: I'm not sure whether this needs to be changed and if so, how
   // Note: protecting this endpoint because we assume in this branch the PA-app will be removed
```
Comment (aberonni) [inline services/121-service/src/programs/programs.controller.ts:60]: See comment above

File: services/121-service/src/fsp/dto/update-fsp.dto.ts (hunk 2 above)
Comment (ashishkumarbasak-accenture) [reply services/121-service/src/fsp/dto/update-fsp.dto.ts:64]: I thin changes are fine in this line, the purpose of this text is just to show it in swagger documentation.

File: services/121-service/src/notifications/lookup/lookup.service.ts [hunk 4]
```diff
@@ -57,19 +57,16 @@ export class LookupService {
...
-              : phoneNumber.substring(0, 1) == '+'
-                ? phoneNumber.substring(1)
-                : phoneNumber;
+        : phoneNumber.substring(0, 3) == '+00'
+          ? phoneNumber.substring(3)
+          : phoneNumber.substring(0, 2) == '+0'
+            ? phoneNumber.substring(2)
+            : phoneNumber.substring(0, 1) == '+'
+              ? phoneNumber.substring(1)
+              : phoneNumber;
```
Comment (ashishkumarbasak-accenture) [inline services/121-service/src/notifications/lookup/lookup.service.ts:69]: what the purpose of removing this condition:
```
phoneNumber.substring(0, 2) == '06'
          ? '31' + phoneNumber
```

File: services/121-service/src/programs/programs.controller.ts (hunk 3 above)
Comment (ashishkumarbasak-accenture) [reply services/121-service/src/programs/programs.controller.ts:60]: No idea, I couldn't help 😞

**Review by ashishkumarbasak-accenture — COMMENTED** [review]
I have one minor comment, Rest is looking great!

File: services/121-service/src/notifications/lookup/lookup.service.ts (hunk 4 above)
Comment (aberonni) [reply services/121-service/src/notifications/lookup/lookup.service.ts:69]: There was a comment before this code block. It said to remove this condition once the PA-App was removed.

File: services/121-service/src/fsp/dto/update-fsp.dto.ts (hunk 2 above)
Comment (aberonni) [reply services/121-service/src/fsp/dto/update-fsp.dto.ts:64]: Yeah, that part is clear to me. What I'm not sure about is whether it is true that this is now simply display name, or whether it is actually unused in the portal (and therefore should be removed).

**Comment by aberonni** [comment]
Thanks @ashishkumarbasak-accenture 

Maybe @PeterSmallenbroek could you review on the remaining conversations? (Without reviewing the whole PR again).

File: services/121-service/src/fsp/dto/update-fsp.dto.ts (hunk 2 above)
Comment (ashishkumarbasak-accenture) [reply services/121-service/src/fsp/dto/update-fsp.dto.ts:64]: Its fine, As far I know, its simply `FSP display name` which is configurable by program now and display it accordingly in the portal.

File: services/121-service/src/programs/programs.controller.ts (hunk 3 above)
Comment (RubenGeo) [reply services/121-service/src/programs/programs.controller.ts:60]: You can leave it like this. We had a discussion about whether we should add an extra read program permission, but it seems so trivial we decided against it. (there is no secret data in the program and every roles should have that permission to be able to use the platform). Maybe just remove your comment @aberonni

**Review by RubenGeo — APPROVED** [review]
Only remove `// XXX: I'm not sure whether this needs to be changed and if so, how`

**Comment by aberonni** [comment]
@RubenGeo pushed another commit where I also remove `phoneNumberPlaceholder` based on Tijs' comment on the DevOps task

**Pull request merged**